src/gwc/src/main/java/org/geoserver/gwc/dispatch/GwcServiceDispatcherCallback.java

```diff
@@ -66,8 +76,22 @@ public Request init(Request request) {
             return null;
         }
 
-        // storing the current operation
-        GWC_OPERATION.set((String) request.getKvp().get("REQUEST"));
+        // storing the current operation (bridge
```

Comment:
Not sure if something is missing after ``(bridge``.

Reply:
Yep, "bridge with RESTful services", which do not have a "request" KVP param